src/test/lib/ddl/ddl_statement_test.cpp

```diff
@@ -0,0 +1,55 @@
+#include <memory>
+#include <string>
+#include <utility>
+
+#include "base_test.hpp"
+#include "tasks/chunk_compression_task.hpp"
+
+#include "SQLParser.h"
+#include "SQLParserResult.h"
+
+#include "types.hpp"
+#include "hyrise.hpp"
+#include "sql/sql_pipeline.hpp"
+#include "sql/sql_pipeline_builder.hpp"
+#include "sql/sql_plan_cache.hpp"
+
+namespace opossum {
+
+class DDLStatementTest : public BaseTest {
+ protected:
+  void SetUp() override {
+    Hyrise::reset();
+
+    // We reload table_a every time since it is modified during the test case.
+    _table_a = load_table("resources/test_data/tbl/int_float_create_index_test.tbl", 2);
+    Hyrise::get().storage_manager.add_table("table_a", _table_a);
+    auto compression_task_0 = std::make_shared<ChunkCompressionTask>("table_a", ChunkID{0});
+    auto compression_task_1 = std::make_shared<ChunkCompressionTask>("table_a", ChunkID{1});
```

Comment:
You could encode the table first using `ChunkEncoder::encode_all_chunks(_table_a)` and add the encoded table to the storage manager afterward. Is there a certain reason why you want to use `ChunkCompressionTask`s after adding the table to the storage manager?

Reply:
We incorporated that now! We also updated create index, create index node, drop index, and drop index node tests accordingly